app/controllers/article_controller.rb

```diff
@@ -114,14 +127,6 @@ def eds_init
       }
     }
 
-    # Register section fields with show/index presenters to leverage rendering pipeline
-    config.show.sections.each do |_section, fields|
```

Comment:
We'll want to keep this loop, but just remove the `config.add_index_field` on line 121.